docs/v2/spec/2018_Modified_LAR_Data_Dictionary.csv

```diff
@@ -0,0 +1,86 @@
+Field Name,Field Data Type,Valid Values

```

Comment:
For the buckets and enumerated values, please standardize on the use of a delimiter. Some are using a semicolon and others only have a space. Either is acceptable, but it needs to be the same. Please remove 'or' from this as well. We want this to be machine readable.

I'd like to not use 'doubles' or 'integers' as valid values. I understand what you mean but have a concern that people will think those are coded values. Let's leave it out (as the FIG does) when the value is a range such as with income.

Reply:
Hi David - addressed your comments above in the latest commit. I decided to leave out the semicolons. I also took out doubles and integers as valid values.